python/setup.cfg

```diff
@@ -1,2 +1,42 @@
-# This file contains the default option values to be used during setup. An
-# example can be found at https://github.com/pypa/sampleproject/blob/master/setup.cfg
+[metadata]
+name = graphframes
+version = 0.8.5
+description = GraphFrames: Graph Processing Framework for Apache Spark
+long_description = file: ../README.md
+long_description_content_type = text/markdown
+author = GraphFrames Contributors
+author_email = graphframes@googlegroups.com
+url = https://pypi.org/project/graphframes-py/
+license = Apache License 2.0
+classifiers =
+    Development Status :: 4 - Beta
+    Programming Language :: Python :: 3
+    Operating System :: OS Independent
+
+[options]
+packages = find:
+package_dir =
+    = python
+include_package_data = True
+install_requires =
+    pyspark>=2.0.0
+    click==8.1.8
+    numpy>=1.7
+    py7zr==0.22.0
+    requests==2.32.3
```

Comment:
It seems to me that this should be a part of dev / additional dependencies

Reply:
@SemyonSinchenko Okay, so I could do a `tutorials` extras. I can do that. It is more than one package it adds... alright :)